api/src/main/scala/com/lightbend/lagom/internal/api/Path.scala

```diff
@@ -7,57 +7,55 @@ import java.util.regex.Pattern
 
 import akka.util.ByteString
 import com.lightbend.lagom.javadsl.api.Descriptor.{ CallId, NamedCallId, PathCallId, RestCallId }
-import com.lightbend.lagom.javadsl.api.deser.RawId.PathParam
-import com.lightbend.lagom.javadsl.api.deser.{ RawId, RawIdDescriptor }
-import org.pcollections.{ PSequence, PMap, HashTreePMap, TreePVector }
 import play.utils.UriEncoding
 
-import scala.collection.JavaConverters._
 import scala.util.parsing.combinator.JavaTokenParsers
 
-case class Path(parts: Seq[PathPart], queryParams: Seq[String]) {
+case class Path(pathSpec: String, parts: Seq[PathPart], queryParams: Seq[String]) {
```

Comment:
Or rather, import scala.collection.immutable and use immutable.Seq as type
to avoid having to look at imports to figure out if it is the right Seq.

-- 
Cheers,
√
On May 3, 2016 9:34 AM, "Mirco Dotta" <notifications@github.com> wrote:

In api/src/main/scala/com/lightbend/lagom/internal/api/Path.scala
<https://github.com/lagom/lagom/pull/99#discussion_r61846650>:

>  import scala.util.parsing.combinator.JavaTokenParsers
>
> -case class Path(parts: Seq[PathPart], queryParams: Seq[String]) {
> +case class Path(pathSpec: String, parts: Seq[PathPart], queryParams: Seq[String]) {

Please add import scala.collection.immutable.Seq everywhere Seq is used.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub
<https://github.com/lagom/lagom/pull/99/files/c52fed6dd29dc6db875f155172543b9a53de4ec4#r61846650>